### PR TITLE
Update README.md with minor fix

### PR DESCRIPTION
**Pull request by harnitsignalfx**
Fix the syntax for adding tags via a dictionary in the trace annotation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,8 +256,8 @@ class MyBusinessLogic:
 
     @staticmethod
     @trace('MyOtherOperation',  # Specify span operation name and tags
-           dict(tag_name='tag_value',
-                another_tag_name='another_tag_value'))
+           dict({tag_name='tag_value',
```
Comment (rmfitzpatrick) [inline README.md:259]: This is invalid syntax: https://docs.python.org/3/library/stdtypes.html#dict

**Comment by harnitsignalfx** [comment]
oops, sorry. Need to use ':' instead. 
Making this change because the currently referenced way does not allow you to set span tags with a period in them. E.g. - 

>>> a = dict('span.kind'='server')
  File "<stdin>", line 1
SyntaxError: keyword can't be an expression
>>> a = dict({'span.kind':'server'})
>>> a
{'span.kind': 'server'}

File: README.md [hunk 2]
```diff
@@ -256,8 +256,8 @@ class MyBusinessLogic:
 
     @staticmethod
     @trace('MyOtherOperation',  # Specify span operation name and tags
-           dict(tag_name='tag_value',
-                another_tag_name='another_tag_value'))
+           dict({tag_name:'tag_value',
```
Comment (rmfitzpatrick) [inline README.md:259]: nit, still invalid syntax and now making two dictionaries.  How about using the named param to make clearer as well? `tags={'tag_name': ...`

Comment (harnitsignalfx) [reply README.md:259]: sorry, might've missed something -  I'd still end up with a single dictionary - dict({tag_name:'tag_value',another_tag_name:'another_tag_value'}) ?

**Comment by harnitsignalfx** [comment]
Going to close this and re-open a clean PR.

**Pull request closed without merging**